draft-looker-oauth-attestation-based-client-authentication.md

```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
 
 --- back
 
+# Additional Examples
+
+## Wallet Instance Attestation
+
+This non-normative example shows a client attestations used as an wallet instance attestation in the context of eIDAS 2.0, e.g. to secure a Type-1 configuration credential. The additional claims describe the wallet's device binding und user binding capabilities and the achievable level of assurance.
+
+~~~
+{
+	"typ": "wallet-attestation+jwt",
+	"alg": "ES256",
+	"kid": "1"
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
```

Comment:
```suggestion
	"iss": "https://wallet.example.com",
```

Reply:
I would rename this to attestation-service.com to signalize that these could be different?

Reply:
I would like to keep the example as simple as possible.  

I think this is a special case, which required explanation. If you provide text, I could incorporate into https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/pull/52